LocalSettings.php

```diff
@@ -6179,7 +6179,7 @@ function onSiteNoticeAfter( &$siteNotice, $skin ) {
 	return true;
 }
 */
-
+/*
 // leaving here as placeholder for future db work
 if ( $wgDBname == 'airwiki' ||
```

Comment:
Remove all these but keep two for examples please.

Reply:
or remove them all because we already have a site notice thing commented out